Upgrade certbot version to 0.31.0

fapdash opened the pull request:
The ppa doesn't offer the 0.28.0 version anymore, so the install failed.

fapdash committed:
Upgrade certbot version to 0.31.0

The ppa doesn't offer the 0.28.0 version anymore, so the install failed.

enricostano commented:
Thanks @fapdash and welcome to the Coopdevs community!

Just one question, does this change play nicely with all the 3 supported platforms?
https://github.com/coopdevs/certbot_nginx/blob/master/meta/main.yml#L10

fapdash commented:
@enricostano I've added the missing platforms to https://github.com/coopdevs/certbot_nginx/pull/15 (https://github.com/coopdevs/certbot_nginx/pull/15/commits/d0f4259e7c67f743ccc2a5cba728543a8f57739e) and the tests ran through.
I guess it was a bit overkill to open three PRs for this, but I wasn't sure if you'd like all the changes I did, so I split it up.

Thank you for the friendly welcome. :)
